### PR TITLE
Update PipeWire to 0.3.29

### PR DESCRIPTION
**Pull request by D3SOX**
https://github.com/PipeWire/pipewire/blob/master/NEWS
0.3.29 was released

### CODE REVIEW

**Review by wizard-28 — APPROVED** [review]
@pacstall/pipewire-git needs to review it

File: packages/pipewire/pipewire.pacscript [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name="pipewire"
 pkgname="pipewire"
```
Comment (wizard-28) [inline packages/pipewire/pipewire.pacscript:2]:
```suggestion

```
@Henryws remove `pkgname` as its a useless field

**Comment by D3SOX** [comment]
> @pacstall/pipewire-git needs to review it

Perhaps rename this team to `pipewire` since the package name changed with #13

**Comment by wizard-28** [comment]
> > @pacstall/pipewire-git needs to review it
> 
> Perhaps rename this team to `pipewire` since the package name changed with #13

@Henryws needs to do this. As he is the maintainer of that team.

**Review by Henryws — COMMENTED** [review]
Looks good

**Comment by Henryws** [comment]
I'll fix the team thing

**Pull request merged**